homeassistant/components/wolflink/sensor.py

```diff
@@ -79,6 +105,19 @@ def extra_state_attributes(self):
             "parent": self.wolf_object.parent,
         }
 
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return the device info."""
+        return DeviceInfo(
+            identifiers={
+                # Serial numbers are unique identifiers within a specific domain
+                (DOMAIN, self._device_id)
+            },
+            name=self._device_name,
+            configuration_url="https://www.wolf-smartset.com/",
+            manufacturer=MANUFACTURER,
+        )
```

Comment:
You could also add `serial_number` to `DeviceInfo`, based on your comment this should be there.

Reply:
The wolf API does not expose the serial number of the device

Reply:
Well, then I misunderstood the comment above referring serial numbers

Reply:
Well, I should probably remove it. It's from the example code sippet